models/lido_liquidity/kyberswap_ethereum/lido_liquidity_kyberswap_ethereum_schema.yml

```diff
@@ -0,0 +1,64 @@
+version: 2
+
+models:
+  - name: lido_liquidity_kyberswap_ethereum
```

Comment:
keep in mind to update schema file contents if naming standards of the spell change